src/admin/components/extensive/Modal/ModalApprove.tsx

```diff
@@ -66,17 +88,13 @@ const ModalApprove: FC<ModalApproveProps> = ({
         const excludedFromValidationCriterias = [COMPLETED_DATA_CRITERIA_ID, ESTIMATED_AREA_CRITERIA_ID];
         const nonValidCriteriasIds = criteria?.nonValidCriteria?.map(r => r.criteria_id);
         const failingCriterias = nonValidCriteriasIds?.filter(r => !excludedFromValidationCriterias.includes(r));
-        let canBeApproved = false;
-        if (criteria?.nonValidCriteria?.length === 0) {
-          canBeApproved = true;
-        } else if (failingCriterias?.length === 0) {
-          canBeApproved = true;
-        }
+        let approved = checkCriteriaCanBeApproved(criteria as ValidationCriteria);
```

Comment:
This can be a `const` now after this refactor

Reply:
changed